A66-otel-stats.md

```diff
@@ -199,21 +204,61 @@ public static class OpenTelemetryModuleBuilder {
      *         .setMeterProvider(sdkMeterProvider)
      *         .build();
      *
+     *     GrpcOpenTelemetry grpcOpenTelemetry = GrpcOpenTelemetry.newBuilder()
+     *         .sdk(openTelemetry)
+     *         .build();
+     *
      * If MeterProvider is not configured, no-op meterProvider will be used by default.
      * It provides meters which do not record or emit.
      */
-    public OpenTelemetryModuleBuilder openTelemetry(OpenTelemetry openTelemetry);
-
-    public OpenTelemetryModule build();
+    public Builder sdk(OpenTelemetry openTelemetry);
+
+    public GrpcOpenTelemetry build();
+  }
+
+  /**
+   * Creates an empty builder.
+   */
+  public static Builder newBuilder();
+
+  /**
+   * Establishes GrpcOpenTelemetry instance as the global instrumentation provider for gRPC opentelemetry,
+   * automatically applying its configuration to all gRPC channels and servers created after this call.
+   *
+   * Sample
+   *    GrpcOpenTelemetry grpcOpenTelemetry = GrpcOpenTelemetry.newBuilder()
+   *         .sdk(openTelemetry)
+   *         .build();
+   *
+   *    grpcOpenTelemetry.registerGlobal();
+   *
+   * <p> Note: Only one of GrpcOpenTelemetry instance can be registered globally. Any subsequent call to
+   * registerGlobal() will throw an IllegalStateException.
+   */
+  public void registerGlobal();
+
+  /**
+   * Configures the given ManagedChannelBuilder with OpenTelemetry metrics instrumentation.
+   */
+  public void configureChannelBuilder(ManagedChannelBuilder<?> builder);
+
+  /**
+   * Configures the given ServerBuilder with OpenTelemetry metrics instrumentation.
+   */
+  public void configureServerBuilder(ServerBuilder<?> serverBuilder);
 }
 ```
 
-Note: For non-generated methods, method names are recorded as "other" for
+Note:
+- For non-generated methods, method names are recorded as "other" for
 `grpc.method` attribute. If you are interested in recording the method names for
 these methods, set
 [`isSampledToLocalTracing`](https://grpc.github.io/grpc-java/javadoc/io/grpc/MethodDescriptor.html#isSampledToLocalTracing\(\))
 to `true` while defining your methods in
 [`HandlerRegistry`](https://grpc.github.io/grpc-java/javadoc/io/grpc/HandlerRegistry.html).
+- A single `GrpcOpenTelemetry` instance can be registered either globally or on a
```

Comment:
why is the behavior unexpected?

Reply:
It collects and exports duplicated data. In OpenCensus writing identical metric more than once throws identical time series error.
Since the identical time series issue has not been verified in OpenTelemetry, updated the note to duplicated data.